explainability-core/src/test/java/org/kie/trustyai/metrics/drift/fouriermmd/FourierMMDTest.java

```diff
@@ -138,4 +182,55 @@ void testProductionData() {
             e.printStackTrace();
         }
     }
+
+    @Test
+    void testRandomData() {
+        try {
+            // def test_tab_prod_data(self):
+            // window = 20
+            // ss = Fourier_MMD(time_column=None, gamma=4, n_window=window,
+            // delta_stat=False)
+
+            // self.train_tab_x = self._generateRandomDataframe(100, 100)
+
+            Dataframe trainTabX = FourierMMDTest.generateRandomDataframe(100, 100);
+
+            // _ = ss.learn(self.train_tab_x)
+
+            final boolean deltaStat = false;
+            final int n_test = 100;
+            final int n_window = 20;
+            final double sig = 10.0;
+            final int randomSeed = 1234;
+            final int n_mode = 512;
+            final double epsilon = 1.0e-7;
+            FourierMMD fourierMMD = new FourierMMD(trainTabX, deltaStat, n_test, n_window, sig, randomSeed, n_mode,
+                    epsilon);
+
+            // d_res = ss.execute(self.test_tab_x)
+
+            // self.test_tab_x = self._generateRandomDataframeDrifted(100,100)
+
+            Dataframe testTabX = FourierMMDTest.generateRandomDataframeDrifted(100, 100);
+
+            final double threshold = 0.8;
+            final double gamma = 4.0;
+            HypothesisTestResult drift = fourierMMD.calculate(testTabX, threshold, gamma);
+
+            // self.assertIsNotNone(d_res)
+            // self.assertIsInstance(d_res, dict)
+            // self.assertListEqual(
+            // list(d_res.keys()), ["drift", "magnitude", "computed_values"]
+            // )
+            // self.assertTrue(d_res["drift"])
+            // self.assertGreaterEqual(d_res["magnitude"], 1)
+
+            Assertions.assertTrue(drift.isReject(), "drifted flag is false");
+
+            Assertions.assertTrue(drift.getpValue() >= 1.0, "drift.pValue < 1.0");
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
```

Comment:
same here, no need to specifically catch the exception

Reply:
fixed